ArchiSteamFarm/Actions.cs

```diff
@@ -294,6 +226,73 @@ internal sealed class Actions : IDisposable {
 			return (true, Strings.BotAutomaticIdlingNowResumed);
 		}
 
+		internal async Task<(bool Success, string Output)> SendTradeOffer(ulong targetSteamID = 0, uint appID = Steam.Asset.SteamAppID, byte contextID = Steam.Asset.SteamCommunityContextID, IReadOnlyCollection<Steam.Asset.EType> wantedTypes = null, IReadOnlyCollection<uint> wantedRealAppIDs = null) {
```

Comment:
Since `targetSteamID` is non-mandatory (can be `0`), while `appID` and `contextID` can't, it should be declared as 3rd argument.